spec/index/index_spec.rb

```diff
@@ -70,6 +70,40 @@
     end
   end
 
+  context '#sort' do
+    let(:idx1) { Daru::Index.new ['speaker', 'mic', 'guitar', 'amp'] }
+    let(:idx2) { Daru::Index.new [:one, 'one', 1, 2, :two] }
```

Comment:
Yes, if you want to show how different indexes behave, you should do it with `context`.
Like this:

```ruby
context '#sort' do
  subject { index.sort(params) }
  context 'when String index' do
    let(:index) { Daru::Index.new ['speaker', 'mic', 'guitar', 'amp'] }
    it { is_expected.to eq Daru::Index.new ['amp', 'guitar', 'mic', 'speaker'] }

    context 'descending' do
      let(:params) { {ascending: false} }
      it { is_expected.to eq Daru::Index.new ['speaker', 'mic', 'guitar', 'amp'] }
    end
  end
...
```